newIDE/app/src/ExportAndShare/ShareDialog/ExportLauncher.js

```diff
@@ -449,13 +472,17 @@ export default class ExportLauncher extends Component<Props, State> {
       authenticatedUser,
       exportPipeline.onlineBuildType
     );
+    const buildCreditPrice = getBuildCreditPrice(
+      authenticatedUser,
+      exportPipeline.onlineBuildType
+    );
 
-    const getNumberOfBuildsCurrentlyRunning = () => {
-      if (!builds) return 0;
+    const hasBuildsCurrentlyRunning = () => {
```

Comment:
I feel like this could be a memoized variable

Reply:
I wish but this is not a functional component, so no straightforward way in a class component